src/components/Cart/Bill.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+function Bill () {
+  return (
+    <div className='h-full md:w-1/3 mob:mt-4 max-w-3xl container  mx-auto p-3  bg-white flex flex-col md:items-top '>
+      <h1 className='font-medium text-black-500 text-lg text-left ml-1'>Bill Details</h1>
```

Comment:
For Bill Details text, remove class `ml-1`. Also on Firefox, this text is not bold like on Chrome. Please check